tensorflow/lite/kernels/test_util.h

```diff
@@ -250,6 +250,7 @@ class SingleOpModel {
   // Build the interpreter for this model. Also, resize and allocate all
   // tensors given the shapes of the inputs.
   void BuildInterpreter(std::vector<std::vector<int>> input_shapes,
+                        int num_threads = -1,
```

Comment:
So, the problem with default args and types is implicit conversion. I think there's already a reference in nnapi_delegate_test to this method, but it silently compiles because bool can be converted to int. You'll need to update that test.

Alternatively, have 3 overloads, 1 for num_threads, 1 for allow_fp32_relax_to_fp16, 1 with just input_shapes, and have them forward to an internal method that takes all 3.

Reply:
@jdduke : Nice catch, i missed in my search i guess, Thanks!